SPECS/local-path-provisioner/local-path-provisioner.spec

```diff
@@ -0,0 +1,36 @@
+Summary:        Provides a way for the Kubernetes users to utilize the local storage in each node
+Name:           local-path-provisioner
+Version:        0.0.21
```

Comment:
Nit: can we use the latest release version (0.0.22)? 

https://github.com/rancher/local-path-provisioner/releases

Reply:
0.0.21 is the version required by the versions of k3s and k8s that we're currently using:
https://dev.azure.com/mariner-org/mariner/_git/mariner-AKSIOT?path=/container-image-configs/k3s/k3s-1.23.6.yaml&version=GBmain&line=7&lineEnd=8&lineStartColumn=1&lineEndColumn=1&lineStyle=plain&_a=contents